test/lib/processors/minifier.js

```diff
@@ -200,10 +206,84 @@ test();`;
 
 	const expected = `//@ui5-bundle sap/ui/my/module.js
 function test(t){var o=t;console.log(o)}test();
-//# sourceMappingURL=test.js.map`;
+${SOURCE_MAPPING_URL}=test.js.map`;
+	t.deepEqual(await resource.getString(), expected, "Correct minified content");
+});
+
+test("addSourceMappingUrl=false", async (t) => {
+	const content = `
+//@ui5-bundle sap/ui/my/module.js
+function test(paramA) {
+	var variableA = paramA;
+	console.log(variableA);
+}
+test();`;
+
+	const testResource = resourceFactory.createResource({
+		path: "/test.js",
+		string: content
+	});
+	const [{resource}] = await minifier({
+		resources: [testResource],
+		options: {
+			addSourceMappingUrl: false
+		}
+	});
+
+	const expected = `//@ui5-bundle sap/ui/my/module.js
+function test(t){var o=t;console.log(o)}test();`;
+	t.deepEqual(await resource.getString(), expected, "Correct minified content");
+});
+
+test("addSourceMappingUrl=true", async (t) => {
+	const content = `
+//@ui5-bundle sap/ui/my/module.js
+function test(paramA) {
+	var variableA = paramA;
+	console.log(variableA);
+}
+test();`;
+
+	const testResource = resourceFactory.createResource({
+		path: "/test.js",
+		string: content
+	});
+	const [{resource}] = await minifier({
+		resources: [testResource],
+		options: {
+			addSourceMappingUrl: true
+		}
+	});
+
+	const expected = `//@ui5-bundle sap/ui/my/module.js
+function test(t){var o=t;console.log(o)}test();
+${SOURCE_MAPPING_URL}=test.js.map`;
 	t.deepEqual(await resource.getString(), expected, "Correct minified content");
 });
 
+test("empty options object (addSourceMappingUrl defaults to true)", async (t) => {
+	const content = `
+//@ui5-bundle sap/ui/my/module.js
```

Comment:
should we add there some indentation?

Reply:
answered above https://github.com/SAP/ui5-builder/pull/709#discussion_r823602439